src/Tizen.NUI/src/public/Theme/ThemeManager.cs

```diff
@@ -126,23 +126,24 @@ public static bool ApplyBaseTheme(string themeId)
 
         /// <summary>
         /// Load a style with style name in the current theme.
-        /// For components, the style name is a component name (e.g. Button) in normal case.
+        /// For components, the style name is a component name with namespace (e.g. Tizen.NUI.Components.Button) in normal case.
```

Comment:
by "in normal case" you mean that it is usually that name, or by convention that name, not the lower/upper case, right?

Reply:
I edited the description. Please check.